core/sql/regress/executor/EXPECTED130

```diff
@@ -124,7 +134,16 @@ inserted row13
 
 --- 3 row(s) selected.
 >>
->>
+>>values (filetolob('./myfile.txt'));
+
+*** ERROR[8442] Unable to access ExpLOB interface. Call to ExpLOB returned error LOB_PTR_ERROR(532). Error detail 0.
```

Comment:
This error will be reutrned anytime an invalid LOB handle is provided to any LOB interface. Before this fix (in ExpLOB.cpp) the mxosrvr could abend. There was no LOB handle provided in this case. So this general error is supposed to indicate this function has no LOB to work on .  If this was a SQL statement referring to actual columns, the LOB column would have provided a LOB handle to for this function to work on.I felt it's better to  return the general error rather than a specific error for the values clause .I can add the causes in the error messages manual.